init.go

```diff
@@ -17,7 +17,6 @@ import (
 	"github.com/aws/shim-loggers-for-containerd/logger/awslogs"
 	"github.com/aws/shim-loggers-for-containerd/logger/fluentd"
 	"github.com/aws/shim-loggers-for-containerd/logger/splunk"
-
```

Comment:
Let's keep the AWS dependencies separate from 3rd party packages

Reply:
Yes will upadte